script/yaml_generator/create_scheduler.py

```diff
@@ -28,19 +28,17 @@ def _get_conf(jobid, srv):
     client = OpenQA_Client(server='http://%s' % srv)
     conf = client.openqa_request('GET', 'jobs/%s' % jobid)
 
-    testname = conf['job']['test']
+    testname = conf['job']['settings']['TEST_SUITE_NAME']
     jobvars = {}
     jobvars['name'] = testname
     suitevars = client.openqa_request('GET', 'test_suites')
 
     vars_generator = (settings for settings in suitevars['TestSuites'])
 
     for v in vars_generator:
-        # there are test jobs that the name is modified in job group and is run with
-        # another name than the name in 'TestSuites' ex installer_extended_textmode
-        # Because ususally the name just appends the name we just check if it startswith
-        if testname.startswith(v['name']):
+        if testname == v.get('name'):
             jobvars.update(v)
+            break
```

Comment:
Are you sure you need this `break` ?

Otherwise LGTM

Reply:
I would like to exit the loop as soon as the job is found, no need to continue to parse it if the job is found.
But if there is a more elegant way to do it in Python that's fine.

Reply:
Then it's fine.